### ISSUE TITLE
Network check crashes Calamares on hasInternet in QML modules

### ISSUE THREAD

**Issue by demmm** [calamares/calamares#1511]
**Describe the bug**
Enabling the Localeq module will crash Calamares after PR https://github.com/calamares/calamares/pull/1502 was merged. 
Crash can be circumvented by eliminating the hasInternet call in https://github.com/calamares/calamares/blob/calamares/src/modules/localeq/localeq.qml#L37 and set that file to load Map.qml instead.
Reverting the above linked commit also fixes the crash.
From the debug output, relevant lines:
```
QML component complete "qrc:/localeq.qml" 
04:15:37 [0]: QQmlEngine: Illegal attempt to connect to CalamaresUtils::Network::Manager(0x2404100) that is in a different thread than the QML engine QQmlEngine(0x2572df0.
KCrash: Application 'calamares' crashing...
KCrash: Attempting to start /usr/lib/drkonqi
```

**To Reproduce**
Enable localeq instead of locale

This bug is not a high priority, because:
 1. Not many use any QML module
 2. Workarounds are pretty easy, either revert one commit, or do not use the hasInternet call in QML modules.

**Comment by adriaandegroot** [comment]
Taking it off the milestone because I want to get things released, and I can't reproduce this on other systems with "just" a config with welcomeq and localeq, so there's something special going on :disappointed:

**Comment by adriaandegroot** [comment]
Fixed in 3.3.0-alpha5
